src/databricks/labs/ucx/queries/assessment/main/05_0_jobs.sql

```diff
@@ -1,7 +1,7 @@
 -- viz type=table, name=Jobs, columns=failure,job_id,job_name,creator
--- widget title=Incompatible jobs, row=5, col=3, size_x=3, size_y=8
+-- widget title=Jobs Details, row=5, col=3, size_x=3, size_y=8
 SELECT
-    EXPLODE(FROM_JSON(failures, 'array<string>')) AS failure,
+    EXPLODE_OUTER(FROM_JSON(failures, 'array<string>')) AS failure,
```

Comment:
Add a new widget with a full list of jobs, otherwise it'll clutter the dashboard

Reply:
Added new widget to display the list of jobs. Also edited the query names to match with row and columns.